components/AddLocation/index.js

```diff
@@ -0,0 +1,257 @@
+import Link from "next/link";
+import Image from "next/image";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import { CldUploadButton, CldImage } from "next-cloudinary";
+
+export default function AddLocation({
+  onSubmit,
+  menuTypes,
+  handleFilter,
+  checked,
+  onOptionChange,
+  imageUrl,
+  setImageUrl,
+  noRental,
+  placeholderImage,
+setImageHeight,
+setImageWidth
+}) {
+ 
+function onUpload(event){
+    setImageUrl(event.info.secure_url)
+    setImageHeight(event.info.height)
+    setImageWidth(event.info.width)
+}
+
+  return (
+    <>
+      <form onSubmit={onSubmit}>
+        <label htmlFor="title"> Location:</label>
```

Comment:
The `htmlFor` attribute doesn't match any existing `id` of input fields.
Currently, the textarea has an `id` of "name" and `htmlFor` has a value of "title".

Reply:
Thank you very much! All fixed!